tools/ci/actions/check-configure.sh

```diff
@@ -25,6 +25,15 @@
 
 set -e
 
+WANT="$(sed -ne 's/^AC_PREREQ(\[\(.*\)\])/\1/p' configure.ac)"
+HAVE=$(sh configure -V | grep "^generated by GNU Autoconf" | awk '{print $NF}')
+if [ "$WANT" != "$HAVE" ]; then
+  echo -ne '\e[31mWrong Autoconf version: \e[0m '
+  echo -e "Have: \e[31m$HAVE\e[0m  Want: \e[32m$WANT\e[0m"
+  echo -e "Run \`tools/autogen autoconf[-]\e[32m$WANT\e[0m\`"
```

Comment:
I don't think it needs colour or backticks for the command (I'm still a too old-fashioned with the backticks - it reads like a shell completion rather than the intended markdown)
```suggestion
  echo -e "Run tools/autogen autoconf[-]$WANT"
```